src/amo/components/WrongPlatformWarning/index.js

```diff
@@ -95,37 +96,88 @@ export class WrongPlatformWarningBase extends React.Component<InternalProps> {
       message = null;
     } else if (newLocation === getMobileHomepageLink(lang)) {
       // Redirecting to mobile home page.
-      message = i18n.sprintf(
-        i18n.gettext(
-          `To find add-ons compatible with Firefox for Android,
-               <a href="%(newLocation)s">click here</a>.`,
-        ),
-        { newLocation },
-      );
+      message = replaceStringsWithJSX({
+        text: i18n.gettext(`To find extensions compatible with Firefox for
+          Android, %(linkStart)sclick here%(linkEnd)s.`),
+        replacements: [
+          [
+            'linkStart',
+            'linkEnd',
+            (text) => (
+              <Link
+                to={newLocation}
+                prependClientApp={false}
+                prependLang={false}
+              >
+                {text}
+              </Link>
+            ),
+          ],
+        ],
+      });
     } else if (addon && newLocation) {
       // User with desktop browser looking at detail page on mobile site.
-      message = i18n.sprintf(
-        `This listing is not intended for this platform.
```

Comment:
This string was never translated because `i18n.gettext()` was missing :/

Reply:
😬